whatshap/core.pyx

```diff
@@ -49,20 +49,23 @@ cdef class NumericSampleIds:
 	def inverse_mapping(self):
 		"""Returns a dict mapping numeric ids to sample names."""
 		return {numeric_id:name for name, numeric_id in self.mapping.items()}
-	
```

Comment:
Please avoid unrleated whitespace-only changes (fixing the trailing whitespace in this file should be a separate PR).

Reply:
fixed